helpers/filterSeats.js

```diff
@@ -60,19 +69,73 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
```

Comment:
I'm really not a fan of a function that take no arguments & modifies variable outside of it's scope. I see this is called a few lines later like `getBestSections()`.

Could we re-factor to become `var bestSections = getBestSections( params.bandColours )` maybe?